presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -753,6 +753,14 @@ The following procedures are available:
     file system paths to use lowercase (e.g. ``col_x=SomeValue``). Partitions on the file system
     not conforming to this convention are ignored, unless the argument is set to ``false``.
 
+* ``system.invalidate_directory_list_cache()``
+
+    Flush full directory list cache.
+
+* ``system.invalidate_directory_list_cache(directory_path)``
```

Comment:
what is the directory_path? How can i build it knowing eg. only a table name in metastore? Could you please provide examples?

Reply:
@mrmaddx2 Here directory_path refers to the path which we would like to exclude from the directory list cache. Right now current procedure doesn't allow us to specify `table name` directly, may be we can add that as an extension if needed in a different procedure call.

Reply:
Could you please provide an example?

Reply:
Sure here ,
`CALL <catalog-name>.system.invalidate_directory_list_cache('<directory_path>')`

If your Hive catalog is called hive.
**Ex :** CALL hive.system.invalidate_directory_list_cache('s3://testTable/partition1')